code/modules/mob/living/carbon/human/human.dm

```diff
@@ -399,6 +399,7 @@
 		dat += "<BR><A href='?src=\ref[src];item=tie'>Remove accessory</A>"
 	dat += "<BR><A href='?src=\ref[src];item=splints'>Remove splints</A>"
 	dat += "<BR><A href='?src=\ref[src];item=pockets'>Empty pockets</A>"
+	dat += species.handle_strip("\ref[src]")
```

Comment:
I am not a fan of that implementation.
Here you use a species proc to determine the text that should be displayed.